### PR TITLE
Mostly consistency but not proportionally as much as last time

### PR DESCRIPTION
**Pull request by spellboundtriangle**
Please read the changes and share your thoughts if you disagree with any of them or found any issues with my wording and such.

### CODE REVIEW

File: pages/_en-US/ds-index/dsi-twl-firm.md [hunk 1]
```diff
@@ -18,31 +18,29 @@ While most of the benefits of Custom Firmware are for the Nintendo DSi and the N
...
+nds-bootstrap has TWL Clock Speed as an option, but it will not try to adjust the ROM to work properly with the higher clock speed. That's on the application itself, and applications not working with a higher clock speed is NOT a bug on the nds-bootstrap end.
 
 ### Nintendo DSi Menu
-In version 1.4.0, RSA signatures in the DS cart whitelist aren't verified. There is an exploit regarding a vulnerability in the Nintendo DSi flashcard whitelist that allows you to take access over the ARM9 processor, It requires version 1.4.0 (it was patched in future versions and didn't exist in prior versions) and a flashcard with a modified ROM.
+In version 1.4.0, RSA signatures in the DS game card whitelist aren't verified. This is a vulnerability that can be exploited, and it allows you to take access over the ARM9 processor. It requires version 1.4.0 (it was patched in future versions and didn't exist in prior versions) and a flashcard with a modified ROM.
```
Comment (lifehackerhansol) [inline pages/_en-US/ds-index/dsi-twl-firm.md:26]: Game Card should be capitalized properly if we're going to use that term

File: pages/_en-US/ds-index/retail-roms.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ Nintendo DS cartridges have 4 known save types:
 - FRAM - Ferroelectric Random-Access Memory
 - NAND - NOT-AND
 
-Though rare, there are DS Cartridges with NAND based saves: WarioWare DIY & Jam with the Band (USA)/Daigasso Band Brothers
+Though rare, there are DS game cards with NAND based saves: WarioWare DIY & Jam with the Band (USA)/Daigasso Band Brothers.
```
Comment (lifehackerhansol) [inline pages/_en-US/ds-index/retail-roms.md:29]: Same as above

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]
If there's any other mentions of Game Cards those should also be proper noun'd

**Comment by spellboundtriangle** [comment]
Done, and edited contributing page for that.

**Pull request merged**